src/Classes/Executor.php

```diff
@@ -124,10 +126,28 @@ public function ping(string $url, array $headers = []): self
     /**
      * Handle the running of a console command.
      *
-     * @param  string  $commandToRun
+     * @param string $commandToRun
+     * @param bool $isInteractive
      */
-    private function runCommand(string $commandToRun): void
+    private function runCommand(string $commandToRun, bool $isInteractive = false): void
     {
+        if ($isInteractive) {
+            // This function is similar to exec(), However, it sends the raw output from the program to the output stream
+            // with which PHP is currently working (i.e. either HTTP in a web server scenario, or the shell in a command line version of PHP).
```

Comment:
It might be nice to extract the interactive command execution stuff into a new method to keep the code split up. Maybe something along the lines of:

```php
if ($isInteractive) {
    return $this->runInteractiveCommand($commandToRun);
}

// Existing code stays here for running non-interactive commands.
```